pom.xml

```diff
@@ -129,7 +129,7 @@
     <hadoop.version>2.7.0</hadoop.version>
     <spark.version>2.4.0</spark.version>
     <scala.binary.version>2.11</scala.binary.version>
```

Comment:
i meant should this be changed to `2.13`?

Reply:
removed it from the root level